### PR TITLE
LPS-194981 Merge PasswordPolicyPermission interface and impl into Util

### PR DESCRIPTION
**Pull request by julschong**
resubmitted from https://github.com/liferay-core-infra/liferay-portal/pull/5492

### CODE REVIEW

File: modules/apps/portal-template/portal-template-engine-api/src/main/java/com/liferay/portal/template/engine/TemplateContextHelper.java [hunk 1]
```diff
@@ -700,8 +700,8 @@ protected void populateCommonHelperUtilities(
 
 		try {
 			variables.put(
-				"passwordPolicyPermission",
-				PasswordPolicyPermissionUtil.getPasswordPolicyPermission());
+				"passwordPolicyPermissionUtil",
```
Comment (julschong) [inline modules/apps/portal-template/portal-template-engine-api/src/main/java/com/liferay/portal/template/engine/TemplateContextHelper.java:703]: added Util suffix to the key

**Comment by julschong** [comment]
ci:test:password-policies

**Comment by Kvale1733** [comment]
@julschong No regressions found.

**Comment by julschong** [comment]
changed variable name back in TemplateContextHelper

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
@shuyangzhou moved to impl

**Comment by shuyangzhou** [comment]
@julschong once you push new commits, please remove the "QA Passed" label to let QA re-review it for the new results.

**Comment by catarineaguiar** [comment]
no regressions found. SF issue is not related

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/141271

**Pull request closed without merging**